pom.xml

```
@@ -455,8 +466,13 @@
 
     <dependencies>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

The `junit-jupiter-api` should also be declared as a direct dependency in addition to the engines.
```suggestion
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifiactId>junit-jupiter-api</artifactId>
            <scope>test</scope>
        </dependency>
        <dependency>
```